README.md

```diff
@@ -1,2 +1,27 @@
-# factory-trytond
-Factory Boy - Trytond integration
+# Factory-trytond
+
+Factory-trytond is a [factory_boy](https://factoryboy.readthedocs.io/en/latest/introduction.html) extension developed to work with [Tryton ERP](https://www.tryton.org/). We can create our own ERP's model factories to do some testing or to populate our databases with some sample data.
+
+## How does it work?
+
+Steps to use Factory-trytond:
+1. Inherit the "base factory" class.
+2. Define the "meta model" the factory "will construct".
+3. Define the "default declarations" of the factory.
```

Comment:
Ookay. With the quotes I was only emphasizing what key concepts I thought should be used to rephrase this section. If written like this I'd drop all the quotation marks